### DISCUSSION TITLE
Other Languages are Possible?

### DISCUSSION THREAD

**Discussion by Veria70** [voicepaw/so-vits-svc-fork#692] [Q&A] [answered]
Hi Guys,
Could you give me road way to training on other languages? like French or Arabic, Difference between Phenomena and Language Character.
Best Regards.

**Comment by Grownz** [comment, marked as answer]
I guess that already works. I read in a chapter of Harry Potter in german as source and generated me singing a french chanson out of it ...
ofc the more the language are phonetically apart the more difficult the result to achieve.

**Comment by erikdarcsi** [comment]
Erikdarcsi123

**Comment by erikdarcsi** [comment]
<img width="2556" height="3408" alt="17900168101833672791621034102543" src="https://github.com/user-attachments/assets/8a9f2c53-658f-4aee-a758-b729b3feff99" />
